### ISSUE TITLE
Updating developer version of Landlab via command line

### ISSUE THREAD

**Issue by amanaster2** [landlab/landlab#875]
Hi all,

I went to the [Updating Landlab](https://github.com/landlab/landlab/wiki/Updating-Landlab) page to see if there was any information about how to update the dev version of Landlab from the command line... The page is somewhat unclear, and I don't think that the git commands on that page help to update the dev version in that way. Are there plans to update the page to include how to update the dev version from the command line?

Cheers,
Amanda

**Comment by kbarnhart** [comment]
@amanaster2 thanks for letting us know that the Updating Landlab page was not sufficient. 

If you want to update your fork's version of `master` here are some more extensive directions:
https://help.github.com/articles/syncing-a-fork/

Once you've done that, you'll want to merge the updated version of `master` into your development branch (assuming you are developing on a development branch). 

Let me know if this helps or if there are other questions I can answer. 

Once we get you happily updated, I'd love if you could provide recommendations on how we should improve the Updating Landlab page.

**Comment by amanaster2** [comment]
@kbarnhart I figured out how to update it based on my limited knowledge of git on the command line. I'll include the process I used to update it... It may not be the most sophisticated, but it worked!

After directing to the folder in which I have my copy of landlab and switching to my `master` branch:
$ `git remote -v` (check which my remotes I have)
`origin https://github.com/amanaster2/landlab.git (fetch)`
`origin https://github.com/amanaster2/landlab.git (push)`
`upstream https://github.com/landlab/landlab.git (fetch)`
`upstream https://github.com/landlab/landlab.git (push)`

$ `git pull upstream master` (fetch & merge from upstream and deal with any conflicts, etc.)
$ `git push origin master` (push to my origin)

I then repeated these steps on my dev branch.

I think that some sort of step-by-step how-to (or even the link to the article you gave me above!) would be a great addition to the Updating Landlab page.

**Comment by kbarnhart** [comment]
@amanaster2 Thanks so much for the comments and letting me know how you solved it. 

Its a little hard to provide super specific step-by-steps, because if a developer on a fork develops on `master` instead of on a `dev` branch then the instructions don't always work. Similarly if the remotes aren't configured correctly people run into issues.

I've added the link (and a link about checking/configuring remotes). I also removed a link that didn't lead to anywhere useful. Let me know what you think (and close the issue if I've made sufficient changes).

Thanks again for pointing this out!
